src/OmniSharp/Api/Highlighting/OmnisharpController.Highlighting.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Text;
+using OmniSharp.Models;
+
+namespace OmniSharp
+{
+    public partial class OmnisharpController
+    {
+        [HttpPost("highlight")]
+        public async Task<IEnumerable<HighlightResponse>> Highlight(HighlightRequest request)
+        {
+            var document = _workspace.GetDocument(request.FileName);
```

Comment:
Apparently my Roslyn/OmniSharp knowledge is missing here. What do you mean handle documents in multiple projects? Is it like if you have one file that's included in multiple projects?